.chloggen/elasticsearchexporter_mapping-hint.yaml

```diff
@@ -7,15 +7,18 @@ change_type: enhancement
 component: elasticsearchexporter
 
 # A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
-note: Emit _doc_count for metric documents in OTel mode when data point attribute _doc_count is true
```

Comment:
[to reviewer] Removing changelog for previous doc count PR as this PR supersedes it. It does not make sense for the doc count PR changelog to be in the release notes of the next version if it is not relevant.